libs/api/domains/license-service/src/lib/client/driving-license-client/drivingLicenseService.api.ts

```diff
@@ -158,9 +163,51 @@ export class GenericDrivingLicenseApi
     return licenses
   }
 
+  private async getCachedGetPkPassToken(): Promise<string | null> {
+    if (!this.cacheManager) {
+      return null
+    }
+
+    const cached = await this.cacheManager?.get(CACHE_KEY)
```

Comment:
```suggestion
    const cached = await this.cacheManager.get(CACHE_KEY)
```

It's already establised that `cacheManager` is defined